libs/utils/utils.js

```diff
@@ -595,11 +595,24 @@ export async function loadDeferred(area, blocks, config) {
 }
 
 export function loadPrivacy() {
+  const domains = {
+    'adobe.com': '7a5eb705-95ed-4cc4-a11d-0cc5760e93db',
+    'hlx.live': '926b16ce-cc88-4c6a-af45-21749f3167f3',
+    'hlx.page': '3a6a37fe-9e07-4aa9-8640-8f358a623271',
+  };
+  const currentDomain = Object.keys(domains)
+    .find((domain) => window.location.host.includes(domain)) || domains[0];
+  let domainId = domains[currentDomain];
+  // Load Privacy in test mode to allow setting cookies on hlx.live and hlx.page
```

Comment:
Cookies cannot be set on `.hlx.live` or `.hlx.page` since they are on the main [suffix list](https://github.com/publicsuffix/list/blob/dd5a45d85e7f23b5e50d02791d9d2e118e36c18d/public_suffix_list.dat#L10671-L10678).
Adding `-test` when Privacy gets initialized will force the OneTrust scripts to set cookies on the page (e.g. main--milo--adobecom.hlx.page) instead of `.hlx.page`. Having the cookies is critical for functionality and end to end testing.